components/TARGET_PSA/TARGET_MBED_SPM/COMPONENT_SPE/psa_setup.c

```diff
@@ -1006,7 +1006,7 @@ void smoke_tests_part1_init(spm_partition_t *partition)
 
 /****************** SPM DB initialization *************************************/
 spm_partition_t g_partitions[] = {
-    /* ATTEST_SRV */
+/* ATTEST_SRV */
```

Comment:
is this again a problem in the script? I recall a fix there

Reply:
@jainvikas8 You might know this one.

Reply:
I rebased to fix this conflict to progress with this PR

Reply:
I think this was fixed in https://github.com/ARMmbed/mbed-os/pull/12085

Reply:
I've checked the base, this branch was based on master from today so should contain the fix but the script still changed the file.

Reply:
If you don't want this to happen, you need https://github.com/ARMmbed/mbed-os/pull/12257